.github/actions/simeng_benchmarks/action.yml

```diff
@@ -2,48 +2,45 @@ name: simeng-benchmarks
 description: runs simeng benchmarks
 
 inputs:
-  BENCHMARK_BRANCH:
-    description: name of the branch on the benchmark repo you want to checkout 
+  RUN_DIR:
+    description: directory from which the benchmark binary should be run
+    required: false
+  DATA_PATH:
+    description: location of the data file(s) for this benchmarks
+    required: false
```

Comment:
As per https://github.com/UoB-HPC/SimEng/pull/437#discussion_r1810502735  I think this should be more generic. Rather than passing just the data path, pass all parameters to be passed to the binary under simulation